src/main/java/invtweaks/InvTweaksConfigManager.java

```diff
@@ -215,8 +217,9 @@ private boolean loadConfig() {
         }
     }
 
-    private boolean extractFile(ResourceLocation resource, File destination) {
-        try(InputStream input = mc.getResourceManager().getResource(resource).getInputStream()) {
+    private boolean extractFile(String resourcePath, File destination) {
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        try (InputStream input = classloader.getResourceAsStream(resourcePath)) {
```

Comment:
Tested it out right now and the language files might work as expected, not sure because Forge is crashing with a NPE.
It tries to access the default configuration and tree-xml, but is not able to be find it on the specified resource path.